docs/api-reference/data-fetching/get-initial-props.md

```diff
@@ -54,7 +54,7 @@ For the initial page load, `getInitialProps` will run on the server only. `getIn
 
 ## Context Object
 
-`getInitialProps` receives a single argument called `context`, it's an object with the following properties:
+`getInitialProps` receives a single argument called `context.ctx`. It's an object with the following properties:
```

Comment:
context.ctx only exists for _app. For pages this object is the parameter of the function.

Reply:
@timneutkens, what do you suggest? Should we explicitly mention that the `context.ctx` exists only for `_app` and for individual pages, it can be used as a parameter of the function?

Reply:
I'd probably just leave it off and mention it in the _app docs